meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit.bb

```diff
@@ -7,6 +7,7 @@ INHIBIT_DEFAULT_DEPS = "1"
 DEPENDS = "dtc-native u-boot-mkimage-native"
 
 SRC_URI = "file://boot.cmd \
+	file://boot.cmd.in \
```

Comment:
I find using 'boot.cmd.in' a bit confusing at this scope, so I would probably change to something more specific to what we're doing here, kind of a boot-common.cmd.in or something similar.

That way it is easier to understand what is going on here, and will help debugging later on.

Reply:
yes, agree. I was hesitating about the filename so I left the most simple one expecting that comment :)